src/main/java/org/dcsa/ovs/notifications/controller/NotificationEndpointController.java

```diff
@@ -0,0 +1,38 @@
+package org.dcsa.ovs.notifications.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.dcsa.core.controller.ExtendedBaseController;
+import org.dcsa.ovs.notifications.model.NotificationEndpoint;
+import org.dcsa.ovs.notifications.service.NotificationEndpointService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.server.reactive.ServerHttpRequest;
+import org.springframework.web.bind.annotation.*;
+import reactor.core.publisher.Mono;
+
+import java.util.UUID;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(value = "unofficial/notification-endpoints", produces = {MediaType.APPLICATION_JSON_VALUE})

```

Comment:
We don't need to call this "unofficial" any more

Reply:
Done